pl_bolts/utils/arguments.py

```diff
@@ -41,10 +41,10 @@ def __init__(self, *args, ignore_required_init_args=True, **kwargs):
         super().__init__(*args, **kwargs)
         self.ignore_required_init_args = ignore_required_init_args
 
-        self._default_obj_args = dict()
-        self._added_arg_names = []
+        self._default_obj_args: Dict[str, List[LitArg]] = dict()
```

Comment:
This is completely off topic and not a strong opinion, but we should use `{}` instead of `dict()` to instantiate an empty dictionary as I remember `{}` is faster than `dict()`.

@hassiahk Would you be interested in submitting a seperate PR replacing `dict()` with `{}` in the codebase?

Reply:
Sure, I can do this.

Reply:
@akihironitta, the entire codebase or just the `pl_bolts.utils.*`?

Reply:
@hassiahk  Please do to the whole codebase in a single PR :]

Reply:
@hassiahk Look forward to reviewing your PR on this :]